ee/codegen_integration/fixtures/simple_map_node/code/display/workflow.py

```diff
@@ -30,9 +30,6 @@ class WorkflowDisplay(VellumWorkflowDisplay[Workflow]):
     )
     inputs_display = {
         Inputs.items: WorkflowInputsVellumDisplayOverrides(id=UUID("cdc4468f-45e7-46ce-bbe7-d1aa9ad86514")),
-        Inputs.items: WorkflowInputsVellumDisplayOverrides(id=UUID("b8d66997-444e-4409-b315-5bef0c06192a")),
```

Comment:
These don't belong here. There is a difference between the Map Node's inputs, and the underlying Subworkflow's inputs. 

We're likely putting it all together in both places in codegen, hence the duplicated `items` and there being 4 inputs in both places.